tests/unit/test_job.py

```diff
@@ -2744,7 +2734,10 @@ def test_cancel_w_bound_client(self):
         final_attributes.assert_called_with({"path": PATH}, None, job)
 
         conn.api_request.assert_called_once_with(
-            method="POST", path=PATH, query_params={}, timeout=None,
+            method="POST",
+            path=PATH,
+            query_params={},
+            timeout=None,
```

Comment:
(this is probably your IDE making changes): don't make whitespace-only changes too far from your main changes. It makes the file history more difficult to read and makes this PR longer. (If you want to clean up the whitespace, I'd suggest making a separate whitespace-only PR for easy reviewing.)

Reply:
I suspect we may need to pin our version of `black`, as different versions can produce different formatting output.

Reply:
Yes I think has been generating new changes to other files. So pining this version is definitely best.

Reply:
Mailed https://github.com/googleapis/python-bigquery/pull/247

Reply:
waiting until #247 is merged to fix this. As black is fixing lines irrelevant to change.